packages/core/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../config/tsconfig.workspace.json",
   "files": [],
   "references": [
-    { "path": "./config/tsconfig.sources.json" },
-    { "path": "./config/tsconfig.test.json" }
+    { "path": "./src/tsconfig.json" },
+    { "path": "./test/tsconfig.json" }
```

Comment:
or do we prefer the shorthand

```suggestion
"references": [{ "path": "./src" }, { "path": "./test" }]
```

like we do with [`aa`](https://github.com/LavaMoat/LavaMoat/pull/752/files#diff-5bafdc438b51c268487cdf7d7c9625b4a3e8dde54aaa2742c13e4065bc12b06fR4) and [`allow-scripts`](https://github.com/LavaMoat/LavaMoat/pull/752/files#diff-eb3256b16392cdc709989b19046f3f164c77ed1cf459cc473b1f99b8804b8153R4)?

Reply:
minor consideration pos worth noting RE dev experience:

the longhand traverses IDE paths nicely

<img width="555" alt="Screenshot 2023-11-09 at 4 29 12 pm" src="https://github.com/LavaMoat/LavaMoat/assets/1881059/a209420b-e1e9-4f06-9e86-4622c5ee862c">

<img width="533" alt="Screenshot 2023-11-09 at 4 30 04 pm" src="https://github.com/LavaMoat/LavaMoat/assets/1881059/395881e8-5fe2-414a-9cc7-6dd6aacfcec1">

whereas shorthand breaks

<img width="645" alt="Screenshot 2023-11-09 at 4 30 33 pm" src="https://github.com/LavaMoat/LavaMoat/assets/1881059/894bef14-d795-4c1f-9ab8-03e876296a21">

<img width="530" alt="Screenshot 2023-11-09 at 4 31 05 pm" src="https://github.com/LavaMoat/LavaMoat/assets/1881059/8fd67c54-5d83-4007-9e00-5bcdd68e6967">

Reply:
I think I prefer to be explicit about the `tsconfig.json` file -- simply because cmd-clicking on the string in VSCode will navigate to the file, whereas omitting `tsconfig.json` will not.

Also it's required if your file _isn't_ called `tsconfig.json` for whatever reason.

So I'll normalize that, too.